README.md

```diff
@@ -120,6 +120,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [InfoDevelopers](https://www.infodev.com.np) | Sanepa, Lalitpur | Providing innovative, quality, cost effective IT Services and Software solutions for business process automation and re-engineering
 - [Insight Workshop](https://insightworkshop.io/) | Kamalpokhari, Kathmandu | Software Development Company
 - [IntroCept Nepal](http://nepal.introcept.co/) | Dhobighat, Lalitpur | A concept development company with skills and expertise to facilitate complex business solutions
+- [Innovate Tech](https://innovatetech.io/) | Baluwatar, Kathmandu | Software Development Company
```

Comment:
@PrannavP Thank you for the contribution. However, can you provide a better description for the company. Every company in this list is a software development company in some way.